examples/autocomplete.amp.html

```diff
@@ -570,5 +572,43 @@ <h2>Templates in search context</h2>
             </template>
         </div>
     </form>
+
+    <h2>'Disable' attribute</h2>
+    <form method="post" action-xhr="/form/echo-json/post" target="_blank">
```

Comment:
Can you add an example of using `disabled` to the `validator-amp-autocomplete.html` as well?